app/controllers/api/v2/report_templates_controller.rb

```diff
@@ -173,7 +173,9 @@ def report_data
         if @plan.progress < 1
           head :no_content
         elsif @plan.failure?
-          render json: { errors: @plan.errors }, status: :unprocessable_entity
+          errors = [{ message: _('Report generating has been canceled') }] if @plan.result == :cancelled
```

Comment:
`Generating of the report has been cancelled.` maybe? Otherwise LGTM

Reply:
:dart: yeah, sounds better.

Reply:
Could also be `Report generation was canceled`. Not a native speaker, but I think even `Report generation canceled` could work.

Reply:
I have used that, but @ares pointed out - quite right - `generation` is a misleading to population (parents, children)